suite-native/module-send/src/sendFormThunks.ts

```diff
@@ -87,6 +89,21 @@ export const cleanupSendFormThunk = createThunk(
     },
 );
 
+export const cleanupBitcoinSendFormsThunk = createThunk(
+    `${SEND_MODULE_PREFIX}/cleanupBitcoinSendFormsThunk`,
+    (_, { dispatch, getState }) => {
+        const sendFormDrafts = selectSendFormDrafts(getState());
```

Comment:
just an idea, decide yourself if you like it:

Create a selector that returns only the accounts that support satoshis and have any send form draft. Then you would not have to filter the array inside of this thunk.

Reply:
I don't want to create anything tailored for this since we don't need it elsewhere.

Reply:
What will be the added value of it then? I'm using the same selectors as in `convertSendFormDraftsBtcAmountUnitsThunk` and I don't think it'll ever be used elsewhere. If yes, we can always extract it.

Anyway, if I'm missing something, I'm all ears.